### DISCUSSION TITLE
[Bug]

### DISCUSSION THREAD

**Discussion by Maldadoscura** [Chia-Network/chia-blockchain#11495] [Support]
### What happened?

Starting services
Full Node

no sincronized it gets stuck i need help .

### Version

1.3.5

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by loppefaaret** [comment]
do you see the window opening up, and then a spinning circle that says it is waiting for "full node" ? 
if you could try and locate `debug.log` file under `~\.chia\mainnet\log\` in your users home folder, and attach it to this discussion, i would have a better chance to figure out what is causing it to not load correctly.
